src/gamebryo/gamebryosavegame.cpp

```diff
@@ -354,6 +334,65 @@ bool GamebryoSaveGame::FileWrapper::openCompressedData(int bytesToIgnore)
   }
 }
 
+bool GamebryoSaveGame::FileWrapper::readNextChunk()
+{
+  uint32_t have;
+  uint64_t read = 0;
+  std::unique_ptr<unsigned char[]> inBuffer(new unsigned char[CHUNK]);
```

Comment:
Should use `std::make_unique` - `std::make_unique<unsigned char[]>(CHUNK)` (or something like that). Same for `outBuffer`.

Reply:
Any reason to use `unsigned char` here vs `char`? AFAIK, `char` is the standard underlying type if you want to `reinterpret_cast`, etc., not sure if this is well defined for `unsigned char`.

Reply:
I think I was just following an example of decompressing from a file and merging that with some other unpacking code we have. But you're probably right.